CONTRIBUTORS.md

```diff
@@ -0,0 +1,7 @@
+# Contributors
+
+Adding your GitHub username to this list confirms that you agree to the terms of
+the Contributor License Agreement.
+
+simpleigh
+drichards2
```

Comment:
@drichards2 please could we keep this file sorted? One day there might be hundreds of entries 😄

Reply:
(I'd just do this, but am quite keen that `git blame` has your name against the line!)

Reply:
Mmmm ... what's the advantage of having it sorted? I can't imagine there's any use case for the file other than searching for a known token and even if there were thousands of contributors, presumably we'd never rely on it being sorted for efficiency reasons (?)

Reply:
No technical reason, just to make it easier for humans. Admittedly applying this criterion is making it harder for one human (you!). Happy either way.